eo-maven-plugin/src/main/java/org/eolang/maven/PullMojo.java

```diff
@@ -111,39 +101,6 @@ public void exec() throws IOException {
             tojo.withSource(this.pull(tojo.identifier()).toAbsolutePath())
                 .withHash(new ChNarrow(this.hsh));
         }
-        Logger.info(
```

Comment:
@volodya-lombrozo what's the reason of removing this log?

Reply:
@maxonfjvipon Returned it back